configuration/config2/ValueDef.hpp

```diff
@@ -0,0 +1,29 @@
+/*/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2017 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#ifndef _CONFIG2_VALUEDEF_HPP_
+#define _CONFIG2_VALUEDEF_HPP_
+
+#include <string>
+#include "platform.h"
+
+struct ValueDef {
```

Comment:
Would "AttributeDef" or "NameValueDef" sometimes be more clear?

Reply:
Changed it to NameValue